packages/sdk/src/core/vincent.ts

```diff
@@ -1,32 +1,28 @@
 import { PKPEthersWallet } from '@lit-protocol/pkp-ethers';
+import { LIT_NETWORK } from '@lit-protocol/constants';
 import { LIT_NETWORKS_KEYS } from '@lit-protocol/types';
 import { ethers } from 'ethers';
 import {
   createJWTConfig,
   createPKPSignedJWT,
   createPKPSigner,
-  DecodedJWT,
   decodeJWT,
-  IStorage, Storage,
   verifyJWTSignature,
 } from '../auth';
 import { DelegateeSigs } from '../pkp';
 
 export interface VincentSDKConfig {
-  storage?: IStorage;
   consentPageUrl?: string;
   network?: LIT_NETWORKS_KEYS;
 }
 
 export class VincentSDK {
-  private storage: Storage;
-  private consentPageUrl: string;
-  private network: LIT_NETWORKS_KEYS;
+  private readonly consentPageUrl: string;
+  private readonly network: LIT_NETWORKS_KEYS;
 
   constructor(config: VincentSDKConfig = {}) {
-    this.storage = new Storage(config.storage);
     this.consentPageUrl = config.consentPageUrl || 'https://demo.vincent.com';
```

Comment:
We should change this default URL should be our heyvincent.ai consent page; this domain actually has something parked on it.